configure.ac

```diff
@@ -3740,16 +3754,16 @@ dnl hence CPPFLAGS instead of CFLAGS.
         PY_CHECK_SQLITE_FUNC([sqlite3_trace])
       ])
       PY_CHECK_SQLITE_FUNC([sqlite3_value_double])
-      AC_CHECK_LIB([sqlite3], [sqlite3_load_extension],
+      PY_CHECK_SQLITE_FUNC([sqlite3_load_extension],
         [have_sqlite3_load_extension=yes],
         [have_sqlite3_load_extension=no]
       )
-      AC_CHECK_LIB([sqlite3], [sqlite3_serialize], [
+      PY_CHECK_SQLITE_FUNC([sqlite3_serialize], [
```

Comment:
The third argument to ``PY_CHECK_SQLITE_FUNC([sqlite3_serialize], ...)``  is blank. Wouldn't this set ``have_supported_sqlite3=no`` if sqlite is missing the feature?

Reply:
🤦🏻‍♂️

Reply:
I added a dummy value. Alternatively we could for example add a fourth argument (required=true|false) to PY_CHECK_SQLITE_FUNC.